### PR TITLE
Add MkrIoTCarrier datasheet

### PR DESCRIPTION
**Pull request by marqdevx**
## What This PR Changes
- Convert MKRIoTCarrier datasheet in MD

## What Needs To Be Reviewed
- Review formatting issues on the content

## How To Give Feedback
Please leave your feedback as a [Github review](https://docs.github.com/en/github/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/about-pull-request-reviews). 
You can add comments to specific lines of content / code and ideally use Github's [suggestion feature](https://docs.github.com/en/github/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/commenting-on-a-pull-request#adding-line-comments-to-a-pull-request). 🙏

### CODE REVIEW

**Review by lunnea — APPROVED** [review]

**Comment by marqdevx** [comment]
@sebromero im getting an issue and i could not fix it 🤔 
image

**Comment by lunnea** [comment]
Before we merge, let @SAUMITRAJAGDALE add some changes from Alessandro in there!

**Comment by marqdevx** [comment]
> Before we merge, let @SAUMITRAJAGDALE add some changes from Alessandro in there!

Sure, all yours! 😄

File: content/hardware/01.mkr/03.carriers/mkr-iot-carrier/datasheet/datasheet.md [hunk 1]
```diff
@@ -0,0 +1,344 @@
...
+Now that you have gone through the basics of what you can do with the board you can explore the endless possibilities it provides by checking exciting projects on ProjectHub **[5]**, the Arduino Library Reference **[6]** and the online store **[7]** where you will be able to complement your board with sensors, actuators and more
+
+### Board Recovery 
+All Arduino boards have a built-in bootloader which allows flashing the board via USB. In case a sketch locks up the processor and the board is not reachable anymore via USB it is possible to enter bootloader mode by double-tapping the reset button right after power up.
+
+## Connector Pinouts
+
+![Pinout](assets/Pinout.png)
+
+Detailed information on the IoT Carrier’s pinouts are available in a separate document: [Docs Arduino - MKR IoT Carrier pinout](https://docs.arduino.cc/resources/pinouts/ABX00047-full-pinout.pdf)
```
Comment (jhansson-ard) [inline content/hardware/01.mkr/03.carriers/mkr-iot-carrier/datasheet/datasheet.md:180]:
```suggestion
Detailed information on the MKR IoT Carrier’s pinouts are available in a separate document: [Docs Arduino - MKR IoT Carrier pinout](https://docs.arduino.cc/resources/pinouts/ABX00047-full-pinout.pdf)
```

File: content/hardware/01.mkr/03.carriers/mkr-iot-carrier/datasheet/datasheet.md [hunk 2]
```diff
@@ -0,0 +1,344 @@
...
+### Getting Started - Arduino Web Editor 
+All Arduino boards, including this one, work out-of-the-box on the Arduino Web Editor **[2]**, by just installing a simple plugin.
+
+The Arduino Web Editor is hosted online, therefore it will always be up-to-date with the latest features and support for all boards. Follow **[3]** to start coding on the browser and upload your sketches onto your board.
+
+### Getting Started - Arduino IoT Cloud 
+All Arduino IoT enabled products are supported on Arduino IoT Cloud which allows you to Log, graph and analyze sensor data, trigger events, and automate your home or business.
+
+### Sample Sketches 
+Sample sketches for the Arduino IoT carrier can be found either in the “Examples” menu in the Arduino IDE or in the Opla Arduino website **[4]** and further documentation can be found in Arduino MKR **[8]** which also contains links to the troubleshooting articles.
```
Comment (jhansson-ard) [inline content/hardware/01.mkr/03.carriers/mkr-iot-carrier/datasheet/datasheet.md:167]:
```suggestion
Sample sketches for the Arduino MKR IoT Carrier can be found either in the “Examples” menu in the Arduino IDE or in the Opla Arduino website **[4]** and further documentation can be found in Arduino MKR **[8]** which also contains links to the troubleshooting articles.
```

File: content/hardware/01.mkr/03.carriers/mkr-iot-carrier/datasheet/datasheet.md [hunk 3]
```diff
@@ -0,0 +1,344 @@
...
+* **Safety information**
+    * Class A 
+
+# Contents
+
+## The Board
+
+### Application Examples
+
+**Light controller:** Control your house lightning with the IoT Carrier, using the RGB light sensor. This feature identifies the general amount of light in a room or environment and adapts. If the board is connected to WiFi,  you can manage and control your device remotely on a smartphone via the IoT Cloud.
```
Comment (jhansson-ard) [inline content/hardware/01.mkr/03.carriers/mkr-iot-carrier/datasheet/datasheet.md:94]:
```suggestion
**Light controller:** Control your house lightning with the MKR IoT Carrier, using the RGB light sensor. This feature identifies the general amount of light in a room or environment and adapts. If the board is connected to Wi-Fi,  you can manage and control your device remotely on a smartphone via the IoT Cloud.
```

File: content/hardware/01.mkr/03.carriers/mkr-iot-carrier/datasheet/datasheet.md [hunk 4]
```diff
@@ -0,0 +1,344 @@
...
+**Weather station:** With temperature sensor, pressure sensor, humidity sensor and light sensors,  you can easily use your IoT carrier as a local weather station. The various sensors can via sensors collect statistics about the different variables required to display weather. 
+
+### Accessories
+
+* 18650 Li-ion battery
+* Grove sensors 
+
+### Related Products
+
+* Arduino MKR 1010 (SKU: ABX00023)
```
Comment (jhansson-ard) [inline content/hardware/01.mkr/03.carriers/mkr-iot-carrier/datasheet/datasheet.md:107]:
```suggestion
* Arduino MKR WiFi 1010 (SKU: ABX00023)
```

File: content/hardware/01.mkr/03.carriers/mkr-iot-carrier/datasheet/datasheet.md [hunk 5]
```diff
@@ -0,0 +1,344 @@
...
+* Protective case
+* Arduino MKR Zero
```
Comment (jhansson-ard) [inline content/hardware/01.mkr/03.carriers/mkr-iot-carrier/datasheet/datasheet.md:109]:
```suggestion
* Arduino® MKR Zero
```

File: content/hardware/01.mkr/03.carriers/mkr-iot-carrier/datasheet/datasheet.md [hunk 6]
```diff
@@ -0,0 +1,344 @@
...
+| J2, J3    | Analog Grove Connectors               | J4       | I2C Grove Connector               |
+| J11       | B2B-PH-SM4-TB(LF)(SN) Power Connector | J9, J10  | 1771033 3-pin Relay Connector     |
+| JDIGITAL1 | MKR Power and Digital Pin Connector   | JANALOG1 | MKR AREF and Analog Pin Connector |
+| S1, S2    | EE2-5NU-L 2A 250V Mechanical Relay    | M1       | Buzzer                            |
+| J1        | Micro SD Module                       | J7, J8   | Li-Ion 18650 Holder               |
+
+
+### Power Tree
+
+The Arduino MKR IoT Carrier makes use of the power management features of the Arduino MKR boards to power itself as well as to interface with the Li-Ion battery.
```
Comment (jhansson-ard) [inline content/hardware/01.mkr/03.carriers/mkr-iot-carrier/datasheet/datasheet.md:141]:
```suggestion
The MKR IoT Carrier makes use of the power management features of the Arduino MKR boards to power itself as well as to interface with the Li-Ion battery. 
```

File: content/hardware/01.mkr/03.carriers/mkr-iot-carrier/datasheet/datasheet.md [hunk 7]
```diff
@@ -0,0 +1,344 @@
...
+
+![Power tree](assets/Power_Tree_MKR_IoT_Carrier.png)
+
+### Solution Overview
+
+![Solution overview](assets/solution_overview.png)
+
+The MKR IoT Carrier is also a part of the Oplá kit, that in addition to the IoT carrier contains an 
+
+Arduino MKR WiFi 1010, a motion sensor, moisture sensor, USB cable and a protective case.
```
Comment (jhansson-ard) [inline content/hardware/01.mkr/03.carriers/mkr-iot-carrier/datasheet/datasheet.md:151]:
```suggestion
The MKR IoT Carrier is also a part of the Oplá kit, that in addition to the IoT carrier contains an Arduino MKR WiFi 1010, a motion sensor, moisture sensor, USB cable and a protective case.  
```

File: content/hardware/01.mkr/03.carriers/mkr-iot-carrier/datasheet/datasheet.md [hunk 8]
```diff
@@ -0,0 +1,344 @@
...
+| Company Address | Via Andrea Appiani,25 20900 MONZA (Italy） |
+
+## Reference Documentation
+
+| **Ref**                   | **Link**                                                     |
+| ------------------------- | ------------------------------------------------------------ |
+| Arduino IDE (Desktop)     | https://www.arduino.cc/en/Main/Software                      |
+| Arduino IDE (Cloud)       | https://create.arduino.cc/editor                             |
+| Cloud IDE Getting Started | https://create.arduino.cc/projecthub/Arduino_Genuino/getting-started-with-arduino-web-editor-4b3e4a |
+| Arduino Opla              | https://opla.arduino.cc/                                     |
```
Comment (jhansson-ard) [inline content/hardware/01.mkr/03.carriers/mkr-iot-carrier/datasheet/datasheet.md:333]:
```suggestion
| Arduino® Oplà IoT Kit              | https://opla.arduino.cc/                                     |
```

File: content/hardware/01.mkr/03.carriers/mkr-iot-carrier/datasheet/datasheet.md [hunk 9]
```diff
@@ -0,0 +1,344 @@
...
+| Project Hub               | https://create.arduino.cc/projecthub?by=part&part_id=11332&sort=trending |
+| Library Reference         | https://www.arduino.cc/reference/en/libraries/arduino_mkriotcarrier/ |
+| Online Store              | https://store.arduino.cc/mkr-iot-carrier                     |
+| Arduino MKR               | https://docs.arduino.cc/hardware/mkr-iot-carrier             |
```
Comment (jhansson-ard) [inline content/hardware/01.mkr/03.carriers/mkr-iot-carrier/datasheet/datasheet.md:337]:
```suggestion
| Arduino Documentation               | https://docs.arduino.cc/hardware/mkr-iot-carrier             |
```

**Comment by jhansson-ard** [comment]
@SAUMITRAJAGDALE please sign the CLA so that we can move forward with this PR ⭐

**Pull request merged**